scripts/wait_for_images.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+PR_NUMBER=$1
+
+REPOSITORY=${REPOSITORY:-kyma-project/kyma-environment-broker}
+HEAD_SHA=$(gh pr view $PR_NUMBER --repo $REPOSITORY --json headRefOid --jq '.headRefOid')
+
+if [ -z "$HEAD_SHA" ]; then
+  echo "Failed to get the head SHA of the pull request"
+  exit 1
+fi
+
+echo "Sleeping for 3 minutes to wait for the image build before checking the status"
+sleep 180
```

Comment:
Image builder always runs longer than 3 minutes, so we don't want to call the API before that time.